README.md

```diff
@@ -43,17 +43,22 @@ The indicator is automatically updated. You don't need to do anything.
 
 File Access | Status Bar Preview |
 ----------- | ------------------ |
-Read-only |![Read-only](images/screenshot-readonly.png) 
-Writeable |![Writeable](images/screenshot-writeable.png) 
+Read-only |![Read-only](images/screenshot-readonly.png)
+Writeable |![Writeable](images/screenshot-writeable.png)
 
 ## Available commands
 
 * `File Access: Change File Access`
+* `File Access: Toggle File Access`
 * `File Access: Make Read-only`
 * `File Access: Make Writeable`
 
 ## Available settings
 
+* `ctrl + shift + a` - `File Access: Toggle File Access`
+
+## Available settings
+
```

Comment:
I prefer avoid adding `keybindings` to my extensions. From time to time VS Code itself adds builtin keybindings which may end up colliding.